imageledger/jinja2/results.html

```diff
@@ -8,7 +8,11 @@
 
 {% with %}
   {% set endpoint = '' %}
-  {% include "includes/search-form.html" %}
+  {% if results.items|length == 0 or request.GET.get('advanced', '') != '' %}
```

Comment:
Not important, but I suspect `if results.items` (without the explicit length check) would just work. Similarly, `if request.GET.get('advanced')` (which will return None as the default value).